erpnext/controllers/status_updater.py

```diff
@@ -13,15 +13,17 @@ def validate_status(status, options):
 
 status_map = {
 	"Lead": [
-		["Converted", "has_customer"],
+		["Lost Quotation", "lost_quotation"],
 		["Opportunity", "has_opportunity"],
+		["Quotation", "has_quotation"],
+		["Converted", "has_customer"],
```

Comment:
This should be first option, because if there is existing customer, status should not other than Converted. It is irrelevant to set lead's status as Opportunity / Quotation.